MycroftAI/mycroft-core
# Merge pull request #1 from MycroftAI/dev

Feature: Allow mycroft-speak and mycroft-say-to to accept bash pipes

## Description
Feature: Allow mycroft-speak and mycroft-say-to take bash pipes #2105

## How to test
with mycroft instance running
run cmd: mycroft-speak test
confirm mycroft speaks
run cmd: echo test | mycroft-speak
confirm mycroft speaks

run cmd: mycroft-say-to "hey mycroft"
confirm mycroft acknowledges
run cmd: echo "hey mycroft" | mycroft-speak
confirm mycroft acknowledges

## Contributor license agreement signed?
CLA [X]
